### PR TITLE
Initial maintainer docs

### PR DESCRIPTION
**Pull request by stuartleeks**
Initial maintainer docs to cover the comment bot commands for running tests

### CODE REVIEW

File: maintainers.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Maintainers
+
+This document is targetted at Microsoft maintainers of the AzureTRE project. For information on developing with Azure TRE, see the [TRE Developers docs](https://microsoft.github.io/AzureTRE/tre-developers/)
```
Comment (ross-p-smith) [inline maintainers.md:3]: Why be explicit (remove Microsoft)

**Review by ross-p-smith — APPROVED** [review]

File: maintainers.md (hunk 1 above)
Comment (stuartleeks) [reply maintainers.md]: updated - thanks

**Review by tamirkamara — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+Note: maintainers see the [separate docs](maintainers.md)
```
Comment (martinpeck) [inline README.md:43]: nit: suggest changing to...

```
Note: maintainers should refer to the [maintainers guide](maintainers.md)
```

**Review by martinpeck — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (stuartleeks) [reply README.md]: updated - thanks

Comment (martinpeck) [reply README.md]: (it makes the link a little more accessible to screen readers if the link text description is stronger)

**Review by marrobi — APPROVED** [review]
LGTM

**Pull request merged**